packages/internal/test/debug-demo.test.js

```diff
@@ -0,0 +1,69 @@
+import { makeError, X } from '@endo/errors';
+import test from '@endo/ses-ava/prepare-endo.js';
+import { makeTracer } from '../src/debug.js';
+
+// As a *-demo.test.js, the purpose of this test file is not so much to be an
+// automated regression test as to be something whose console output
+// should be visually and manually inspected.
+//
+// XXX makeTracer should somehow be migrated to the endo repo, perhaps as
```

Comment:
```suggestion
// TODO makeTracer should somehow be migrated to the endo repo, perhaps as
```

per [kinds of comments](https://github.com/Agoric/agoric-sdk/wiki/Coding-Style#code-comments):
> `TODO` notes something to come back and do.

Reply:
Done